CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/templates/RecordStoreDropper.java

```diff
@@ -111,4 +100,31 @@ protected RecordStore query(final RecordStore query, final PluginInteraction int
         return null;
     }
 
+    @PluginInfo(pluginType = PluginType.IMPORT, tags = {"IMPORT"})
+    public static class RecordStoreDropperRecordStoreQueryPlugin extends RecordStoreQueryPlugin {
```

Comment:
the name is a bit confusing to me as it sounds more like a plugin type than a plugin. The previous name included a mention of record store to graph. Could be good to include that somewhere here (unless of course that was wrong)?

Reply:
Thanks and I agree. I updated the name based on what the previous name was and called it `RecordStoreDropperToGraphPlugin`